test/tokens/ZcToken.sol

```diff
@@ -42,8 +42,10 @@ contract ZcToken is Erc20, IERC5095 {
             return 0;
         }
 
-        // NOTE we weren't even using the pre-hydrated redeemer in most of these anyway...
-        return principalAmount * IRedeemer(redeemer).exchangeRate(protocol, cToken) / IRedeemer(redeemer).markets(protocol, underlying, maturity).maturityRate;
+        // note that maturity rate (mRate) == exchangeRate (xRate) if the mRate would have been 0 (see VaultTracker.rates)
+        (uint256 mRate, uint256 xRate) = IRedeemer(redeemer).rates(protocol, underlying, maturity);
+
+        return principalAmount * xRate / mRate;
```

Comment:
Nit: instead of `xRate` can we use the full word (spotRate?), same for `mRate`. Up to you if this change is worth doing, but I think it would help with legibility of the codebase.

Reply:
spotRate does not exist in the codebase and i'm not going to introduce another word. its pretty clearly marked these are maturity rate and exchange rate. 

`maturityRate` would clash with the global. i damn sure aint using `_maturityRate` so `mRate` works for me.

Reply:
i too hate reading mRate and xRate but dont have better solution, and agree that itd be `exchangeRate` and `maturityRate` or an alternative to `maturityRate`